ckan/lib/i18n.py

```diff
@@ -77,6 +75,16 @@ def get_ckan_i18n_dir() -> str:
     return path
 
 
+def get_js_translations_dir() -> str:
+    storage_path = config.get(
+        "ckan.storage_path"
+    ) or tempfile.gettempdir()
```

Comment:
I copied this logic from the [webassets](https://github.com/ckan/ckan/blob/2718a137ecaf2956e471ba1fd4286566800c2861/ckan/lib/webassets_tools.py#L151) handling module (minus the dedicated config options, which seemed overkill). I think that works fine for something semi-ephemeral like these small json files, but maybe at some point we would need to think about requiring a valid and existing value for `ckan.storage_path` to start up, like we require `ckan.site_url` or others.

Reply:
we should avoid predictable temp files by using [mkdtemp](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp) or similar instead of gettempdir (this goes for webassets too)